pkg/upstream/cluster/strict_dns_cluster.go

```diff
@@ -251,6 +251,7 @@ func (rt *ResolveTarget) StartResolve() {
 		case <-rt.stop:
 			rt.resolveTimeout.Stop()
 			rt.resolveTimer.Stop()
+			return
```

Comment:
Thank you for fixing it.
The following [case <-rt.stop](https://github.com/mosn/mosn/pull/1894/files#diff-c36f7a5b9260c86ee2d12759eb6b391caf06b96ed271f4fe52b285ce32ed90e1R257) can also be returned directly, thus reducing a select.